tcav.py

```diff
@@ -239,36 +239,36 @@ def _run_single_set(self, param):
                             cav_hparams.alpha)
     target_class_for_compute_tcav_score = target_class
 
-    for cav_concept in concepts:
-      if cav_concept is self.random_counterpart or 'random' not in cav_concept:
-        i_up = self.compute_tcav_score(
-            mymodel, target_class_for_compute_tcav_score, cav_concept,
-            cav_instance, acts[target_class][cav_instance.bottleneck])
-        val_directional_dirs = self.get_directional_dir(
-            mymodel, target_class_for_compute_tcav_score, cav_concept,
-            cav_instance, acts[target_class][cav_instance.bottleneck])
-        result = {
-            'cav_key':
-                a_cav_key,
-            'cav_concept':
-                cav_concept,
-            'target_class':
-                target_class,
-            'i_up':
-                i_up,
-            'val_directional_dirs_abs_mean':
-                np.mean(np.abs(val_directional_dirs)),
-            'val_directional_dirs_mean':
-                np.mean(val_directional_dirs),
-            'val_directional_dirs_std':
-                np.std(val_directional_dirs),
-            'note':
-                'alpha_%s ' % (alpha),
-            'alpha':
-                alpha,
-            'bottleneck':
-                bottleneck
-        }
+    cav_concept = concepts[0]
```

Comment:
will this error in the case of empty concepts list whereas the previous code would correctly return no results? if so, please fix to exhibit the same behavior in that strange error case.

Reply:
I haven't been able to find a case when this causes an issue (i.e. running no concepts means there are 0 parameters, and so this function is not called), and when I set concepts to the empty list inside the function, it causes issues before this line is reached (specifically in get_or_train_cav).